docs/xtrabackup_bin/restoring_individual_tables.md

```diff
@@ -12,59 +12,47 @@ a int(11) DEFAULT NULL
 
 ## Export the table
 
-Created the table in innodb_file_per_table mode, so
-after taking a backup as usual with the –backup option, the
+Created the table in `innodb_file_per_table` mode, so
+after taking a backup as usual with the --backup option, the
 .ibd file exists in the target directory:
 
 ```{.bash data-prompt="$"}
 $ find /data/backups/mysql/ -name export_test.*
 /data/backups/mysql/test/export_test.ibd
 ```
 
-when you prepare the backup, add the –export option to the
+when you prepare the backup, add the --export option to the
 command. Here is an example:
 
 ```{.bash data-prompt="$"}
 $ xtrabackup --prepare --export --target-dir=/data/backups/mysql/
 ```
 
-!!! note
-   
-    If you restore an encrypted InnoDB tablespace table, add the  keyring file:
+If you restore an encrypted InnoDB tablespace table, add the keyring file:
 
-    ```{.bash data-prompt="$"}
-    $ xtrabackup --prepare --export --target-dir=/tmp/table \
-    --keyring-file-data=/var/lib/mysql-keyring/keyring
-    ```
+```{.bash data-prompt="$"}
+$ xtrabackup --prepare --export --target-dir=/tmp/table \
+--keyring-file-data=/var/lib/mysql-keyring/keyring
+```
 
-Now you should see an .exp file in the target directory:
+Now you should see the `export_test.ibd` and `export_test.cfg` files in the target directory:
```

Comment:
Restructure the information
The following files are the only files required to import the table into a server running Percona Server for MySQL with XtraDB or MySQL 8.0. If the server uses InnoDB Tablespace Encryption, add the .cfp file, which contains the transfer key and an encrypted tablespace key.

The files are located in the target directory:

/data/backups/mysql/test/export_test.ibd
/data/backups/mysql/test/export_test.cfg

Reply:
updated